src/Avalonia.Base/PriorityBindingEntry.cs

```diff
@@ -92,33 +93,52 @@ public void Dispose()
 
         private void ValueChanged(object value)
         {
-            _owner.Owner.Owner?.VerifyAccess();
+            void Signal()
+            {
+                _owner.Owner.Owner?.VerifyAccess();
```

Comment:
Is it really needed to call VerifyAccess here? You've just checked it and scheduled the call back to the UI thread